conf/sha

```diff
@@ -0,0 +1,8 @@
+accumulo:1.8.1:eba3bfe823935ca7901ea7c2bd59c84a68b9381361699c7e260bbd9191f237f4
```

Comment:
Uno supports multiple hash types. calling the single file "sha" is misleading. If a user specifies a different hash type (MD5, for instance), it will still work. I would call the file "hashes" or "checksums"

Reply:
Renamed file to `checksums` in bffdf6f8b2